src/components/profileDetails/PersonProfile.test.js

```diff
@@ -57,4 +57,29 @@ describe('<PersonProfile />', () => {
     fireEvent.click(getByText('DONATE'));
     expect(history.location.pathname).toBe('/donate/identifier');
   });
+
+  test('renders N/A when num_of_children is 0', () => {
+    const history = createMemoryHistory();
+    const noChildrenInfo = info;
+    noChildrenInfo.number_of_children = 0;
```

Comment:
`noChildrenInfo` is assigned `info`'s reference so by doing `noChildrenInfo.number_of_children = 0` you are actually changing the original `info` object.

```suggestion
    const noChildrenInfo = {
       ...info,
       number_of_children: 0,
    };
```